tests/providers/test_bank.py

```diff
@@ -207,3 +210,48 @@ def test_iban(self, faker, num_samples):
             assert is_valid_iban(iban)
             assert iban[:2] == TrTrBankProvider.country_code
             assert re.fullmatch(r"\d{2}\d{22}", iban[2:])
+
+
+class TestDeCh:
+    """Test de_CH bank provider"""
+
+    def test_bban(self, faker, num_samples):
+        for _ in range(num_samples):
+            assert re.fullmatch(r"\d{17}", faker.bban())
+
+    def test_iban(self, faker, num_samples):
+        for _ in range(num_samples):
+            iban = faker.iban()
+            assert is_valid_iban(iban)
+            assert iban[:2] == DeChBankProvider.country_code
+            assert re.fullmatch(r"\d{19}", iban[2:])
+
+
+class TestFrCh:
```

Comment:
Absolutely no need to change this, but I just want to let you know that you could have done something like [this](https://github.com/joke2k/faker/blob/9e360fa90eb6bfe3fde4c320ac32f033cc803e00/tests/providers/test_bank.py#L187) since the 3 locales have the same implementation anyway.